rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -1651,6 +1654,20 @@ pub mod sns {
             };
             Decode!(&result, sns_pb::GetUpgradeJournalResponse).unwrap()
         }
+
+        pub async fn advance_target_version(
```

Comment:
I think you may have some commits mixed into this PR that come from elsewhere?  This doesn't seem to match the description.

Reply:
Thanks, not sure how these changes made it into the PR.